app/components/App/styles.css

```diff
@@ -85,6 +94,21 @@ button {
   width: 60px;
 }
 
+.download-apps {
```

Comment:
General CSS classes should be avoided if possible when app grows the risk of overriding styles is higher. 

I strongly suggest to move the specific css to each folder. For example if the ```DownloadApps/index.js``` uses  ```download-app``,  you should create a `DownloadApps/index.css``` and then inside ```DownloadApps/index.js``` you should call:

```import `index.css```

Once you have redistributed the CSS, next step is achieve "CSS-in-modules", basically, with that technique, you assure not other styles are affecting the specific css.

After it, you should be able to use ```scss``` variables. Keep in mind when designers create proper mocks to you you have to update the views as fast as possible.

For doing the css-in-modules stuff, you have to install different webpack loaders and configure them. I suggest to take a look into ```gnosis-team-safe``` project. There you will find everything (even the configuration for using webpack 4).